src/bitcoin/OrdinalsClient.ts

```diff
@@ -86,34 +86,36 @@ export class OrdinalsClient {
 
   async getMetadata(inscriptionId: string): Promise<Record<string, unknown> | null> {
     if (!inscriptionId) return null;
-    const info = await this.fetchJson<any>(`/inscription/${inscriptionId}`);
-    if (!info) return null;
-
-    // If API provides metadata directly
-    if (info.metadata && typeof info.metadata === 'object') {
-      return info.metadata as Record<string, unknown>;
-    }
-
-    // If metadata is hex-encoded CBOR string
-    if (info.metadata && typeof info.metadata === 'string') {
-      try {
-        const bytes = hexToBytes(info.metadata);
-        return decodeCbor<Record<string, unknown>>(bytes);
-      } catch {
-        // fall through
-      }
-    }
-
-    // Fallback: if content type is CBOR, fetch and decode
-    if (info.content_type === 'application/cbor') {
-      const contentUrl = info.content_url || `${this.rpcUrl}/content/${inscriptionId}`;
-      const res = await fetch(contentUrl);
+    const base = this.rpcUrl.replace(/\/$/, '');
+    const paths = [
+      `/r/metadata/${inscriptionId}`,
```

Comment:
@cursoragent the `/r/metadata/${inscriptionId}` endpoint is the only one that exists